.travis.yml

```diff
@@ -16,6 +16,8 @@ script:
     - make test_coverage
     - flake8 --version
     - make flake8
-    - if [[ "${TRAVIS_PYTHON_VERSION}" != "pypy"* ]] ; then make mypy ; fi
```

Comment:
the 3.4 build was failing because mypy no longer supports it, so this change is to remove 3.4. I couldn't find the correct syntax for doing ORs in Travis, so I resorted to 3 positive IFs.

Reply:
That's fair, I removed 3.4 in master just now so if you rebase your branch this particular change will no longer be necessary.